theano/tensor/c_code/alt_blas_template.c

```diff
@@ -1,8 +1,43 @@
 /** Alternative template NumPy-based implementation of BLAS functions used in Theano. **/
 
+/* Compute tensor[i] = scalar for every position i in tensor. */
```

Comment:
I think the code here only work well for matrix. So rename tensor to matrix in this diff.